### PR TITLE
Project: Newsweek

### PR DESCRIPTION
**Pull request by OlukaDenis**
- Use of bootstrap class
- Responsivity
- Three break points

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1753 @@
...
+              </path>
+            </g>
+          </svg>
+        </a>
+        <div class="col p-0">
+          <div class="row justify-content-end align-content-center text-uppercase font-weight-bold pr-0">
+            <a class="text-decoration-none text-white px-2 pt-2" href="#">Sign in</a>
+            <a class="text-decoration-none px-2 pt-2" style="background: white; color:#f72210; " href="#">Subscribe <i
```
Comment (wathigo) [inline index.html]: **[REQUIRED]** Remove inline styles. Add all your styles to the external CSS file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1753 @@
...
+                class="fas fa-chevron-right pl-1"></i></a>
+            <button class="navbar-toggler p-0 ml-2" type="button" data-toggle="collapse" data-target="#collapsingNavbar">
+              <span class="navbar-toggler-icon pt-2" style="color: white;"></span>
```
Comment (wathigo) [inline index.html]: **[REQUIRED]** Remove inline styles. Add all your styles to the external CSS file.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1753 @@
...
+            </button>
+            <div class="collapse navbar-collapse" id="collapsingNavbar">
+              <ul class="style navbar-nav float-right  mt-3" style="margin-right: -1em; background: white;">
```
Comment (wathigo) [inline index.html]: **[REQUIRED]** Remove inline styles. Add all your styles to the external CSS file.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1753 @@
...
+                  <a class="nav-link text-dark" href="#">Vintage</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    
+    <div class="container-fluid  d-lg-block d-md-none d-sm-none " style="background-color: rgba(244, 245, 246, 0.973);">
```
Comment (wathigo) [inline index.html]: **[REQUIRED]** Remove inline styles. Add all your styles to the external CSS file.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1753 @@
...
+                </path>
+              </g>
+            </svg>
+          </a>
+          <div class="header-buttons">
+            <a class=" text-white  text-decoration-none pr-2 font-weight-bolder" href=" #">SIGN IN</a>
+            <a class=" text-decoration-none font-weight-bolder p-1" style=" background: white; color:rgb(247,34,16)"
```
Comment (wathigo) [inline index.html]: **[REQUIRED]** Remove inline styles. Add all your styles to the external CSS file.

**Review by wathigo — CHANGES_REQUESTED** [review]
### Hi @OlukaDenis  :wave:  
I like the effort you have put into this project.
Let's fix a couple of things before we mark this as complete.

**[REQUIRED]**
* We have a problem at a screen width of `600px`. The **Top Stories** heading is not visible. See :point_down:  
Screenshot from 2019-10-18 15-17-10

* On the smaller screens a hamburger icon appears on the navigation bar.
* At a screen size of around `500px`. The sub-nav is displayed and we have a big margin on the right. The sub-nav should not displayed. Let's fix this :point_down: 
Screenshot from 2019-10-18 15-51-16

* Reduce the margin on the left and right of the `footer` element to look more like the original page on the smaller screens.
* Fix the issues pointed out in the comments below :point_down:  

After implementing the changes, submit another request for review.

_Happy Coding :computer: 
Reviewed by: **Simon Wathigo**_

**Review by Redvanisation — APPROVED** [review]
### Hello @OlukaDenis  🌟 

### _Nice work on this project, it seems that you have met the requirements so I am approving your project. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

**Pull request merged**